admin_app/src/app/dashboard/components/Overview.tsx

```diff
@@ -1,7 +1,95 @@
 import React from "react";
 import { Box } from "@mui/material";
+import { StatCard, StatCardProps } from "@/app/dashboard/components/StatCard";
+import ForumIcon from "@mui/icons-material/Forum";
+import SupportAgentIcon from "@mui/icons-material/SupportAgent";
+import NewReleasesOutlinedIcon from "@mui/icons-material/NewReleasesOutlined";
+import ThumbUpIcon from "@mui/icons-material/ThumbUp";
+import ThumbDownIcon from "@mui/icons-material/ThumbDown";
+import { Period } from "../types";
+import { useAuth } from "@/utils/auth";
+import { getStatsCardData } from "@/app/dashboard/api";
+import { useEffect } from "react";
+
+interface OverviewProps {
+  timePeriod: Period;
+}
+
+const Overview: React.FC<OverviewProps> = ({ timePeriod }) => {
+  const { token } = useAuth();
+  const [statCardData, setStatCardData] = React.useState<StatCardProps[]>([]);
+  useEffect(() => {
```

Comment:
I remember you mentioned components should be dull and thus should not call APIs, should we do the same for this and make the api call inside the page rather than the component?

Reply:
Hmm yeah. That is true for low level components. Let me think about what to do about higher level ones. This is the entire page 1.